x-pack/plugins/fleet/common/types/models/preconfiguration.ts

```diff
@@ -24,6 +24,7 @@ export type InputsOverride = Partial<NewPackagePolicyInput> & {
 
 export interface PreconfiguredAgentPolicy extends Omit<NewAgentPolicy, 'namespace' | 'id'> {
   id: string | number;
+  kibana_namespace?: string;
```

Comment:
`namespace` is already used in, curious if someone has better naming idea for that one

Reply:
`namespace` is generally used more "internally" in Kibana code whereas `space` is used more in the public-facing docs, UI, etc. Maybe `kibana_space_id` would make sense here?

Reply:
> `namespace` is generally used more "internally" in Kibana code whereas `space` is used more in the public-facing docs, UI, etc. Maybe `kibana_space_id` would make sense here?

++ either `kibana_space_id` or `space_id` would be preferred. @kpollich is right that `namespace` is the internal name (for fun historical reasons) that we try not to leak.

Reply:
I will update to `space_id` as it seems more used in other plugins